packages/shared/src/components/widgets/SimilarPosts.tsx

```diff
@@ -36,7 +40,7 @@ export type SimilarPostsProps = {
   };
 };
 
-const Separator = <div className="h-px bg-theme-divider-tertiary" />;
+const HorizontalSeparator = <div className="h-px bg-theme-divider-tertiary" />;
```

Comment:
Would you mind updating this bg?
I just removed all of them in my PR

Reply:
Sure.